src/OmniSharp.Roslyn/WorkspaceServices/ExportWorkspaceServiceFactoryWithAssemblyQualifiedNameAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Composition;
+using System.Reflection;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace OmniSharp
+{
+    [MetadataAttribute]
+    [AttributeUsage(AttributeTargets.Class)]
+    public class ExportWorkspaceServiceFactoryWithAssemblyQualifiedNameAttribute : ExportAttribute
```

Comment:
cc @savpek needed to pull this in 😀
used the same name so that there are no conflicts